style_rwd.css

```diff
@@ -2,21 +2,233 @@
+    }
+    .section_wrapper{
+        max-width:767px;
+        margin:0 auto;
+        padding: 0;
+    }
+    .font_size{
+        font-size:10px;
+    }
+    .section_hero {
+        padding-bottom: 42px;
     }
     .column{
+        max-width:767px;
         align-items: center;
         justify-content: center;
     }
-    .invite_phrase{
+    .catch_phrase .invite_phrase{
         text-align: center;
-        font-size: 5px;
-        font-weight: 30;
-        max-width: 767px;
+        font-size: 40px;
+        margin-bottom: 0;
+        max-width:767px;
+    }
+    .button_container {
+        justify-content: flex-start;
+        max-width:767px;
+        margin: 0 0 0 100px;
     }
-    .btn{
+    .btn {
+        border-radius: 32px;
+        padding: 5px 16px;
         font-size: 12px;
     }
-}
+    .section_container {
+        min-height: 300px;
+        justify-content: center;
+    }
+    .catch_phrase {
+        flex-direction: column;
+        max-width:300px;
+    }
 
+    .flex_column {
+        flex-direction: column;
+    }
+
+    .section_heading {
+        font-size: 18px;
+        max-width:767px;
+        padding: 17px 0 20px;
+    }
+    .pros_photo{
+        height: auto;
+        margin: 0 auto;
+    }
+    .pros_container {
+        max-width:767px;
+        margin-top: 15px;
+    }
+    .text {
+        font-size: 16px;
+        margin-bottom: 16px;
+        max-width: 767px;
+    }
+    .mountains_pic{
+        display:none;
+    }
+    .paragraph_description{
+    margin: 0 0 5px 20px;
+    }
+    .longer_paragraph{
+        max-width: 767px;
+        margin-left: 8px;
+    }
+    .max_width{
+        justify-content: center;
+        align-items: flex-start;
+        max-width: 767px;
+    }
+    .section_center_media{
+        justify-content:flex-start;
+    }
+    .right_side{
+        max-width: 767px;
+        height: auto;
+    }
+    .post_container{
+        text-align: center;
+    }
+    .constant_width{
+        max-width: 300px;
+    }
+    .left_side{
+        margin: 40px 0 0 0;
+        display: block;
+    }
+    .pink{
+        margin: 0;
+        display: block;
+        }
+    .left_side_margin{
+        line-height: 25px;
+    }
+    .invite_phrase{
+        line-height:55px;
+    }
+    .button_book_margin{
+        margin: 0 0 0 70px;
+    }
+    .pic{
+        max-width: 100%;
+    }
+
+    .section_pink_wrapper{
+        padding:0 8px;
+    }
+    .paragraph_header {
+        margin-bottom: 8px;
+        margin-top: 16px;
+    }
+    .pros_con_right {
+        padding:0;
+    }
+    .third_page_header{
+        margin-left: 8px;
+    }
+    .padding{
+        flex-direction: column;
+    }
+    .girl_pic{
+        max-width: 252px;
+        max-height: 302px;
+    }
+    .left_side_fourth_page{
+        height:auto;
+        margin-top: 50px;
+    }
+    .right_side_fourth_page {
+        max-width: 767px;
+        height: auto;
+        margin-left: 16px;
+    }
+    .cabin_lovers{
```

Comment:
remove margin from this class. Open devtools and investigate one child after another until you make sure 
- there are  no negative margins
- highest margin value on x-axis is 32px (or auto)

Reply:
there shouldn't be any negative margin in whole code?

Reply:
that's correct

Reply:
fixed